src/webgl/program.js

```diff
@@ -153,6 +169,11 @@ export default class Program extends Resource {
     drawParams.isIndexed = false;
     drawParams.indexType = null;
 
+    // Reutrn early if no buffers to be bound.
```

Comment:
I would not recommend checking for empty object as a general micro-optimization technique. It is non-idiomatic JavaScript, and I doubt there is much perf to be gained in this case: No buffers should just result in an empty (no op) sort and an empty iteration.

Unless you have numbers showing that this is an issue, I would drop.